modules/auth/login-service.ts

```diff
@@ -0,0 +1,15 @@
+import supabase from '@/services/auth/auth-supabase.service';
+
+export async function registerUser(email, password) {
```

Comment:
This function shall go into the file `auth-supabase.service.ts`  

The modules layer doesn't;t contain any service